src/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -45,7 +45,7 @@ public with sharing class AFFL_MultiRecordTypeMapper {
     /*******************************************************************************************************
     * @description Affiliation Account-record-type-to-Contact-field mappings.
     ********************************************************************************************************/
-    public Map<String, String> mapAccRecTypeToContactLabel;
+    public Map<String, String> AccRecTypeToContactLabel;
```

Comment:
Sorry to be a stickler on this, but we usually start variable names with lowercase, like the ones below. :-)